hathor/client.py

```diff
@@ -392,7 +392,8 @@ def create_tx_from_dict(data: Dict[str, Any], update_hash: bool = False,
     if storage:
         data['storage'] = storage
 
-    cls = TxVersion(data['version']).get_cls()
+    version = data['version'] & 0xFF
```

Comment:
Maybe use a constant here? `TxVersion.VERSION_MASK` and `TxVersion.SIGNAL_BITS_MASK`?

Reply:
Great! Thanks!